extensions/amp-analytics/0.1/test/test-variables.js

```diff
@@ -140,19 +149,16 @@ describes.fakeWin('amp-analytics.VariableService', {amp: true}, env => {
     });
 
     it('expands array vars', () => {
-      check(
-        '${array}',
-        'xy%26x,MACRO(abc,def),MACRO(abc%2Cdef)%26123,%24%7Bfoo%7D',
-        {
-          'foo': 'bar',
-          'array': [
-            'xy&x', // special chars should be encoded
-            'MACRO(abc,def)', // do not encode macro
-            'MACRO(abc,def)&123', // this is not a macro
-            '${foo}', // vars in array is not expanded
-          ],
-        }
-      );
+      check('${array}', 'xy%26x,MACRO(abc,def),MACRO(abc%2Cdef)%26123,bar,', {
```

Comment:
seems we didn't have a test to verify an array w/o vars in it. could you add one?

Reply:
good call